src/doc/rgw/rgw-qos.md

```diff
@@ -0,0 +1,32 @@
+# RGW Quality of Service
```

Comment:
this really about the grouping of QoS response by all the RGWs, right?  there's a larger QoS topic this fits into